gui/tests/test_addons_editor.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import tempfile
+import mozfile
+
+from PyQt4.QtCore import Qt, QStringList
+from mock import patch
+from mozregui.addons_editor import AddonsWidgetEditor
+
+
+@pytest.fixture
+def addons_editor(qtbot):
+    widget = AddonsWidgetEditor()
+    qtbot.addWidget(widget)
+    widget.show()
+    qtbot.waitForWindowShown(widget)
+    return widget
+
+
+def get_view_focus(addons_editor):
+    return addons_editor.ui.list_view.viewport().focusWidget
+
+
+def test_creat_addons_editor(addons_editor):
+    assert addons_editor.get_addons() == []
+
+
+@pytest.fixture
+def addons_file(request):
+    # create a temp addons file
+    f = tempfile.NamedTemporaryFile(suffix='.xpi', dir='.', delete=False)
+    request.addfinalizer(lambda: mozfile.remove(f.name))
```

Comment:
Oh! Sorry, it seems like went wrong in here, but it's wired because all works fine on locally :(